tests/gtest/README

```diff
@@ -0,0 +1,21 @@
+VMA Testing Environment by GTEST README
+
+This set of tests is based on Google Test C++ environment 
+https://github.com/google/googletest
+
+
+
```

Comment:
I wouldn't leave more than a single line difference when you want to separate lines of text. There's no need for more and it looks tidier.